app/src/main/java/com/github/se/project/ui/profile/EditTutorProfile.kt

```diff
@@ -0,0 +1,221 @@
+
+  val academicLevel = remember { mutableStateOf(profile.academicLevel.name) }
+  val section = remember { mutableStateOf(profile.section.name) }
+  var expandedSection by remember { mutableStateOf(false) }
+  var expandedAcademicLevel by remember { mutableStateOf(false) }
+
+  val context = LocalContext.current
+
+  Scaffold(
+      topBar = {
+        IconButton(
+            onClick = { navigationActions.goBack() },
+            modifier = Modifier.testTag("editTutorProfileCloseButton")) {
+              Icon(
+                  imageVector = Icons.AutoMirrored.Default.ArrowBack,
+                  contentDescription = "Go Back")
+            }
+      },
+      content = { paddingValues ->
+        Column(
+            modifier =
+                Modifier.fillMaxSize()
+                    .padding(horizontal = 16.dp)
+                    .padding(paddingValues)
+                    .testTag("tutorInfoScreen"),
+            verticalArrangement = Arrangement.spacedBy(8.dp)) {
+              Text(
+                  text = "${profile.firstName} ${profile.lastName}",
+                  modifier =
+                      Modifier.testTag(
+                          "editTutorWelcomeText"), // .padding(vertical = 48.dp, horizontal =
+                  // 16.dp).testTag("welcomeText"),
+                  style = MaterialTheme.typography.headlineMedium,
+                  textAlign = TextAlign.Center)
+
+              Text(
+                  "Modify your profile information:",
+                  style = MaterialTheme.typography.titleMedium,
+                  modifier = Modifier.testTag("editTutorProfileInstructionText"))
+
+              Spacer(modifier = Modifier.height(16.dp))
+
+              Text(
+                  "Teaching languages:",
+                  style = MaterialTheme.typography.titleSmall,
+                  modifier = Modifier.testTag("editTutorProfileLanguageText"))
+              // Language Selection
+              LanguageSelector(profileLanguages)
+
+              Spacer(modifier = Modifier.height(16.dp))
+
+              // Subject Selection
+              Text(
+                  "Teaching subjects:",
+                  style = MaterialTheme.typography.titleSmall,
+                  modifier = Modifier.testTag("editTutorProfileSubjectText"))
+              SubjectSelector(null, profileSubjects, true)
+
+              Spacer(modifier = Modifier.height(16.dp))
+
+              // Price Selection
+              Text(
+                  "Tutoring price per hour:",
+                  style = MaterialTheme.typography.titleSmall,
+                  modifier = Modifier.testTag("editTutorProfilePriceText"))
+              PriceSlider(priceSliderValue)
+
+              Spacer(modifier = Modifier.height(8.dp))
+
+              Text(text = "Modify your section", style = MaterialTheme.typography.titleSmall)
+              // Section dropdown menu with improved styling
+              Box {
```

Comment:
we should be able to extract that Section selector (line 123 to 152) into a composable since this is the exact same code as in the CreateUserProfile.kt. I'll do it later its fine for now!